modules/wpml/wpml-config.php

```diff
@@ -32,17 +32,6 @@ class PLL_WPML_Config {
 	 */
 	protected $files;
 
-	/**
-	 * Constructor
-	 *
-	 * @since 1.0
-	 */
-	public function __construct() {
-		if ( extension_loaded( 'simplexml' ) ) {
-			$this->init();
-		}
-	}
-
```

Comment:
Are you sure that you did not remove the whole `wpml-config.xml` feature here?

Reply:
oops yes :-(